src/Mod/Part/Gui/CommandFilter.cpp

```diff
@@ -170,10 +170,10 @@ PartCmdVertexSelection::PartCmdVertexSelection()
   : Command("Part_VertexSelection")
 {
     sGroup        = "Standard-View";
-    sMenuText     = QT_TR_NOOP("Vertex Selection");
-    sToolTipText  = QT_TR_NOOP("Select a Vertex/Vertices");
+    sMenuText     = QT_TR_NOOP("Vertex selection");
+    sToolTipText  = QT_TR_NOOP("Select a vertex/vertices");
```

Comment:
Should we drop the singular form? I think if it says "Select vertices" then it should be clear that one can also select a single vertex only.